README.md

```diff
@@ -1,46 +1,76 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Ez Car
+# Ez Car Backend
 
-> This project contains an API that is consumed by our React frontend. It was built using Rails as an API.
+> This is an API allows users to book a car present in our database. The user can set the reservation date and final date for a selected car. If the user is an administrator they can upload a car and delete it from our database.
 
 ## ERD diagram
 
 ![ERD](https://user-images.githubusercontent.com/97900045/196997928-2c857de9-a9fc-4216-8480-9580cc9bc6c0.png)
 
 ## Built With
 
+### Technologies 
+
 - Ruby on Rails
 - PostgreSQL
 - RSpec
 - Rswag
+- Devise 
+- Devise-JWT
+- CanCanCan
+- Capybara
 
-## Live Demo (if available)
+### Tools
 
-[Live Demo Link](https://livedemo.com)
+- Visual Studio Code
+- GIT
+- Rubocop
+- Heroku 
 
+## Live Demo (if available)
 
-## Getting Started
+- [eZ_car](https://ezcar-backend.herokuapp.com/)
```

Comment:
- [ ] For your live demo, please, kindly consider linking to your rswag docs instead of a blank page:

`- [eZ_car](https://ezcar-backend.herokuapp.com/api-docs)`

So instead of this:

![image](https://user-images.githubusercontent.com/57421823/199338173-bbec273a-3802-441f-928a-53e67befaae9.png)

Your users will get this:

![image](https://user-images.githubusercontent.com/57421823/199338325-c856a4e1-eec5-4338-b988-23814320d22e.png)



That way, your swagger docs will serve as a landing point for your API live demo. 💪

Reply:
@Meltrust Thank you for the recommendation. We implemented it and now the link is not empty